distributions/osgi/features/replication/src/main/feature/feature.xml

```diff
@@ -40,6 +40,6 @@
         <bundle>mvn:replication/replication-ui/${project.version}</bundle>
         <bundle>mvn:replication/commons/${project.version}</bundle>
         <bundle>mvn:replication-adapters/ddf-adapter/${project.version}</bundle>
-        <bundle>mvn:replication-adapters/webhdfs-adapter/${project.version}</bundle>
+<!--        <bundle>mvn:replication-adapters/webhdfs-adapter/${project.version}</bundle>-->
```

Comment:
Should this be removed? Or maybe leave a comment about why it's commented out (if HDFS is not supported here)?

Reply:
removed commented out line